pkg/cluster/install.go

```diff
@@ -49,7 +49,13 @@ func (m *manager) AdminUpdate(ctx context.Context) error {
 
 func (m *manager) Update(ctx context.Context) error {
 	steps := []steps.Step{
+		steps.Action(m.initializeKubernetesClients), // must be first
 		steps.AuthorizationRefreshingAction(m.fpAuthorizer, steps.Action(m.validateResources)),
+		// credentials rotation flow steps
+		steps.Action(m.createOrUpdateClusterServicePrincipalRBAC),
+		steps.Action(m.createOrUpdateDenyAssignment),
+		steps.Action(m.updateAROSecret),
+		steps.Action(m.updateOpenShiftSecret),
```

Comment:
What ensures that the cloud controller restarts after this secret changes?  Can we wait for it?